src/app/components/challenge/challengesubmit/challengesubmit.component.ts

```diff
@@ -428,6 +428,9 @@ export class ChallengesubmitComponent implements OnInit {
         self.globalService.setFormValueForLabel(self.components, 'method_description', '');
         self.globalService.setFormValueForLabel(self.components, 'project_url', '');
         self.globalService.setFormValueForLabel(self.components, 'publication_url', '');
+        if (self.validFileUrl === true) {
+        self.globalService.setFormValueForLabel(self.components, 'file_url', '');
```

Comment:
Same for this (https://github.com/Cloud-CV/EvalAI-ngx/pull/242/files#diff-7ebebddad833dbfe4a95492011383387R413)